content/strategy-goals/strategy/code-graph/index.md

```diff
@@ -1,30 +1,28 @@
 # Code Graph strategy
 
-Code Graph comprises teams working on [Search](./search/index.md), [Code Insights](./code-insights/index.md), [Code Intelligence](./code-intelligence/index.md), and [Batch Changes](./batch-changes/index.md); this page describes the mission and strategy that ties them together over the next year or so, but each linked page provides a more detailed strategy for that team. There is also a whole-company [Sourcegraph strategy](../index.md) and [Product & Engineering strategy](../../../departments/product-engineering/strategy-goals/index.md) to which all of this is aligned. Our [roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/34) (internal only) also shows where we are headed over the next year in terms of specific deliverables.
+Code Graph comprises teams working on [Search](./search/index.md), [Code Insights](./code-insights/index.md), [Code Intelligence](./code-intelligence/index.md), and [Batch Changes](./batch-changes/index.md); this page describes the mission and strategy that ties them together over the next year or so, with each linked page providing a more detailed strategy for that area. There is also a whole-company [Sourcegraph strategy](../index.md) and [Product & Engineering strategy](../../../departments/product-engineering/strategy-goals/index.md) to which all of this is aligned. Our [roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/34) (internal only) shows what we're delivering to make this strategy a reality.
 
-If you're new to our strategy pages, this video walkthrough will help orient you.
+If you're new to our strategy pages, this video walkthrough (last updated Feb 2022) will help orient you.
 
 <div style="position: relative; padding-bottom: 56.25%; height: 0;"><iframe src="https://www.loom.com/embed/71b675b67c464b0db7670a7f5025cbbe" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
 
-## Themes
-
-Our mission is to **make discovering, learning, and editing code easier, no matter how complex it is, where it's stored, or how experienced you are.** We will achieve this through several important themes:
-
-### Make the power of our features easier to find and use
+## Mission
 
-Getting the most out of our product can take time as you get up to speed with what's possible. Making it easier to learn will help teams find the value more quickly than today. Our task is to find a way to make interacting with multiple languages and repositories easy to understand.
+Our mission is to **make discovering, learning, and editing code easier, no matter how complex it is, where it's stored, or how experienced you are.**
```

Comment:
```suggestion
Our mission is to **make discovering, analyzing, and editing code easier, no matter how complex it is, where it's stored, or how experienced you are.**
```

This is only soft feedback, but I don't know that "discovering code" (or "learning code") is a good way to describe Code Insights, since it's not really about the code, but the code *signals* (which are often metadata-like) you are trying to unearth. What about this option?

Reply:
```suggestion
Our mission is to **make discovering, learning, analyzing, and editing code easier, no matter how complex it is, where it's stored, or how experienced you are.**
```

I think learning is still a use case (perhaps not for Insights, but for other parts of Code Graph) but analyzing is a great addition!